paymentsheet/src/test/java/com/stripe/android/paymentsheet/SavedPaymentMethodMutatorTest.kt

```diff
@@ -674,11 +666,7 @@ class SavedPaymentMethodMutatorTest {
             )
 
             savedPaymentMethodMutator.setDefaultPaymentMethod(paymentMethods[1])
-
-            val succeededCall = eventReporter.setAsDefaultPaymentMethodSucceededCalls.awaitItem()
-            assertThat(succeededCall).isInstanceOf(FakeEventReporter.SetAsDefaultPaymentMethodSucceededCall::class.java)
-            assertThat(succeededCall.paymentMethodType).isNotNull()
-            assertThat(succeededCall.paymentMethodType).isEqualTo(paymentMethods[1].type?.code)
```

Comment:
I don't think we should remove these validations, unless we have a good reason to.

Reply:
Ohh, my bad. Looks like the new helper is validating them. Nice!